test/callback/test_diagnostic_hdf5_output.py

```diff
@@ -13,6 +13,7 @@ def tmp_outputdir(tmpdir_factory):
     return str(fn)
 
 
+@pytest.mark.xfail
 def test_callbacks(tmp_outputdir):
```

Comment:
This seems to use `tmp_outdir` for the output directory, but then doesn't use that when trying to read things.